scripts/maven/Dependency.kt

```diff
@@ -0,0 +1,19 @@
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class that contains all the details relevant to a dependency that is present
+ * in maven_intall.json.
+ */
+@JsonClass(generateAdapter = true)
+data class Dependency(
```

Comment:
I suggest differentiating between maven_install & dependency_list Moshi classes by prefixing them as so:

MavenListDependency
LicenseDependency
...

WDYT?

Reply:
Sounds good to me, but the refactoring here works only at some places and it requires a little manual work, so can I do it once the script runs correctly?

Reply:
Refactoring done in the fake package.